Build/Common.Build.CSharp.targets

```diff
@@ -19,6 +19,8 @@
       <_NugetAssemblies Include="$(PackagesPath)\Microsoft.VisualStudio.*\lib\net45\Microsoft.VisualStudio.*.dll" />
       <_NugetAssemblies Include="$(PackagesPath)\Microsoft.VisualStudio.*\lib\net46\Microsoft.VisualStudio.*.dll" />
       <_NugetAssemblies Include="$(PackagesPath)\Microsoft.VisualStudio.*\lib\net461\Microsoft.VisualStudio.*.dll" />
+      <_NugetAssemblies Include="$(PackagesPath)\Microsoft.VisualStudio.*\lib\net472\Microsoft.VisualStudio.*.dll" />
```

Comment:
What are these new assembles for?

Reply:
Some of the updated NuGet packages are distributing .NET 4.72 and .NET Standard builds, and since VS 2019 extensions target 4.72, that's what we end up pulling from NuGet. We need those entries in order to find those assemblies.